fixtures/pytest_store.py

```diff
@@ -61,6 +61,7 @@ class Store(object):
     def current_appliance(self):
         # layz import due to loops and loops and loops
         from utils import appliance
+        assert self.config is not None, 'current appliance not in scope'
```

Comment:
**REQUIRED:** I'm scared of adding this in without a specified reason, have we seen this happen somewhere?